docs/src/nucleation_plotting/CLOUD_Nucleation_Plots.jl

```diff
@@ -1,39 +1,13 @@
 using Plots
-using CLIMAParameters
 
-include("../../src/Nucleation.jl")
-using .Nucleation
-
-# Testing for CLOUD-experiment based nucleation rates.
+import CLIMAParameters as CP
+import CloudMicrophysics.Parameters as CMP
+import CloudMicrophysics.Nucleation as Nucleation
+include("../../test/create_parameters.jl")
```

Comment:
Is this way better? Or it doesn't really matter?

include(joinpath(pkgdir(CM), "test", "create_parameters.jl"))